webpack can not found the bundle asset

```
ossinsight-web-1  | No bundles were parsed. Analyzer will show only original module sizes from stats file.
ossinsight-web-1  | 
ossinsight-web-1  | Webpack Bundle Analyzer is started at http://127.0.0.1:8888
ossinsight-web-1  | Use Ctrl+C to close it
ossinsight-web-1  | client (webpack 5.74.0) compiled with 1 error
ossinsight-web-1  | ℹ Compiling Client
ossinsight-web-1  | ✔ Client: Compiled with some errors in 854.24ms
ossinsight-web-1  | 
ossinsight-web-1  | 
ossinsight-web-1  | Module not found: Error: Can't resolve '@query/trending-repos/params.json' in '/app/src/pages/home/_sections/2-toplistv2'
ossinsight-web-1  | Error parsing bundle asset "/app/build/main.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/vendors-node_modules_djagger_echartsx_dist_index_js-node_modules_geo-maps_countries-land-10km-3df424.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/vendors-node_modules_mui_icons-material_People_js-node_modules_mui_icons-material_PlayArrow_j-3bd897.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/vendors-node_modules_docusaurus_core_lib_client_exports_BrowserOnly_js-node_modules_docusauru-dc4cc2.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/vendors-node_modules_makotot_ghostui_dist_ghostui_esm_js-node_modules_mui_icons-material_Code-c999fb.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/vendors-node_modules_primer_octicons-react_dist_index_esm_js-node_modules_param-case_dist_es2-695d6f.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/__comp---site-src-dynamic-pages-analyze-703-0b8.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/vendors-node_modules_mui_icons-material_ArrowDownward_js-node_modules_mui_icons-material_Arro-57c387.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/vendors-node_modules_sql-formatter_lib_sqlFormatter_js.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---blog-deep-insight-into-open-source-databases-348-cf8.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---blog-say-thanks-to-github-robots-129-622.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/vendors-node_modules_docusaurus_plugin-debug_lib_theme_DebugJsonView_index_js-node_modules_do-50b6ef.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/__comp---site-src-pages-2022-index-tsx-9-bf-95d.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---blog-deep-insight-into-js-framework-202124-b-a13.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---blog-deep-insight-into-web-framework-20218-da-63d.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---blog-deep-insight-into-lowcode-development-tools-20210-bb-de4.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---blog-deep-insight-into-programming-languages-20213-d-9-6b2.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---blog-74-f-466.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---blog-918-404.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---blog-763-479.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---blog-8-be-0d8.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---blogd-11-c3e.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---blog-page-26-f-2-7f1.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/__comp---site-src-dynamic-pages-analyze-user-39-e-52d.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/__comp---site-src-pages-index-tsx-1-df-d3e.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---blog-explore-deep-in-4-6-billion-github-eventsc-11-10e.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/vendors-node_modules_docusaurus_theme-classic_lib_theme_EditThisPage_index_js-node_modules_do-6b30a3.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/__comp---site-src-dynamic-pages-collections-8-c-3-617.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/__comp---site-src-dynamic-pages-collections-dynamic-trendsaf-2-cb4.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/allContent---docusaurus-debug-content-246-9aa.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/vendors-node_modules_mui_icons-material_ExpandMore_js-node_modules_mui_material_esm_Accordion-46adf4.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/archive---blog-archiveb-2-f-393.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/vendors-node_modules_mui_icons-material_ArrowDownward_js-node_modules_mui_icons-material_Arro-1f2cc7.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---blog-2-c-2-33e.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---blog-trends-and-insights-from-github-2022-f-9-f-a34.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---blog-1-e-2-d4b.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---blog-reduce-query-latency-802-99d.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/vendors-node_modules_docusaurus_theme-classic_lib_theme_DocBreadcrumbs_index_js-node_modules_-5b696a.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---blogc-82-eb2.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---blog-why-we-choose-tidb-to-support-ossinsighta-3-e-6fc.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/vendors-node_modules_docusaurus_theme-classic_lib_theme_CodeBlock_index_js.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/vendors-node_modules_mui_material_esm_Table_Table_js-node_modules_mui_material_esm_TableBody_-4ba968.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/vendors-node_modules_mui_material_esm_FormControlLabel_FormControlLabel_js-node_modules_mui_m-2f3071.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---docs-workshop-mini-ossinsight-step-by-step-get-insights-with-sqla-8-f-4a2.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/__comp---site-src-dynamic-pages-stats-0-b-8-b99.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/__comp---site-src-pages-try-your-own-dataset-index-tsx-17-a-4a4.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/vendors-node_modules_docusaurus_theme-classic_lib_theme_BlogLayout_index_js-node_modules_docu-e82834.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/src_components_ECharts_lazy_tsx-src_theme_CodeBlock_CopyButton_index_tsx.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---blog-page-2-ac-1-8b9.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---blog-try-it-yourself-777-13b.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---blogcec-5a9.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---blog-github-data-is-booming-8-c-9-4af.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/vendors-node_modules_mui_icons-material_Add_js-node_modules_mui_material_esm_ToggleButton_Tog-53108f.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/__comp---site-src-pages-collections-index-tsx-962-192.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/runtime~main.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/__comp---site-src-dynamic-pages-stats-tablesc-41-387.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---docs-about-3-d-8-538.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/vendors-node_modules_docusaurus_theme-classic_lib_theme_DocPage_index_js.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/vendors-node_modules_docusaurus_theme-classic_lib_theme_DocSidebar_Desktop_CollapseButton_ind-21a38d.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/vendors-node_modules_djagger_echartsx_dist_components_option_series_line_js-node_modules_djag-f76c5e.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---blog-how-it-worksd-15-58d.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/vendors-node_modules_react-transition-group_esm_CSSTransition_js.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/vendors-node_modules_mui_material_esm_Dialog_Dialog_js.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/vendors-node_modules_mui_icons-material_Code_js-node_modules_react-intersection-observer_reac-0aa99e.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/__comp---theme-blog-tags-posts-page-687-b6c.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/vendors-node_modules_mui_icons-material_ArrowDownward_js-node_modules_mui_icons-material_Arro-624f58.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/vendors-node_modules_djagger_echartsx_dist_components_option_title_index_js-node_modules_djag-8cf6d5.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/__comp---theme-blog-post-pageccc-cab.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/vendors-node_modules_mdx-js_react_dist_esm_js-node_modules_mui_material_esm_Divider_Divider_js.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---blog-page-29-d-4-6f7.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---blog-saas-insight-for-building-a-real-time-crm-applicationd-31-b97.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/vendors-node_modules_mui_material_esm_Link_Link_js-node_modules_mui_material_esm_TableContain-053cd5.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/__comp---theme-blog-tags-list-page-01-a-d0b.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/__comp---theme-blog-list-pagea-6-a-7ba.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/__comp---theme-doc-page-1-be-9be.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/vendors-node_modules_mdx-js_react_dist_esm_js-node_modules_docusaurus_theme-classic_lib_theme-0ed8ed.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---docs-workshop-mini-ossinsight-step-by-step-find-data-sourcefbe-dd4.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---docs-workshopee-8-546.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---docs-workshop-cryptocurrency-insightc-93-84a.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/vendors-node_modules_tslib_tslib_es6_js.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/__comp---site-src-pages-analyze-company-index-tsx-908-d94.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/vendors-node_modules_mui_lab_LoadingButton_LoadingButton_js-node_modules_fast-deep-equal_index_js.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---blog-page-2-f-31-b67.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---docs-workshop-twitter-insight-827-83a.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/__comp---site-src-pages-subscriptions-index-tsx-6-f-9-758.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/__comp---theme-doc-item-178-a40.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---docs-workshop-stackoverflow-insight-1-c-5-d62.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---blog-page-2-a-49-758.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---docs-api-endpoints-trending-repos-790-d37.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---docs-workshop-mini-ossinsight-introductionb-0-e-9d1.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/vendors-node_modules_docusaurus_theme-classic_lib_theme_TOC_index_js.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---docs-api-65-d-4a0.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---docs-workshop-mini-ossinsight-step-by-step-load-data-to-tidbeca-973.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---docs-faq-048-db5.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---docs-workshop-nft-insighte-60-d58.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---docs-showcase-27-d-4cb.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---blog-3-ba-4e4.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/content---blog-who-works-the-longest-hours-on-github-847-315.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/__comp---theme-debug-site-metadata-68-e-3d4.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/src_components_Chart_Chart_tsx.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/__comp---theme-debug-registry-679-501.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/src_theme_NotFound_tsx.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/runtime~main.d65b10f3c251666b.hot-update.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/__comp---theme-blog-archive-page-9-e-4-1d8.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/__comp---theme-debug-routes-946-699.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/__comp---site-src-pages-subscriptions-enable-email-switch-tsx-38-c-241.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/versionMetadata---docs-0-b-3-5c5.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/__comp---theme-debug-contentba-8-ce7.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/__comp---theme-debug-config-23-a-2ff.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/__comp---theme-debug-global-dataede-0fa.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/sidebar---blog-814-8ac.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/__comp---site-src-pages-compare-jsxea-1-b54.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/listMetadata---blog-tags-tidbcloudbb-9-174.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/listMetadata---blog-tags-insight-2-d-0-b2d.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/metadata---blog-page-28-eb-625.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/listMetadata---blog-tags-tidb-486-def.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/metadata---blogb-2-b-df1.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/tags---blog-tagsa-70-da2.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/tag---blog-tags-tidbcloude-56-875.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/tag---blog-tags-insightdb-1-743.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/tag---blog-tags-tidbd-85-9a6.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/plugin---2022-ad-7-b8b.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/plugin---blog-574-931.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/plugin---docusaurus-debug-4-d-9-11e.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/plugin---docs-9-aa-778.js": no such file
ossinsight-web-1  | Error parsing bundle asset "/app/build/plugin---analyze-logina-0-a-495.js": no such file
ossinsight-web-1  | 
ossinsight-web-1  | No bundles were parsed. Analyzer will show only original module sizes from stats file.
ossinsight-web-1  | 
ossinsight-web-1  | client (webpack 5.74.0) compiled with 1 error
```

--- a/web/docusaurus.config.js
+++ b/web/docusaurus.config.js
@@ -65,7 +65,7 @@ const config = {
     [
       path.resolve(__dirname, 'plugins/alias'),
       {
-        '@query': path.resolve(__dirname, '../api/queries/')
+        '@query': path.resolve(__dirname, '../configs/queries/')
       }
     ],
     [

--- a/web/docusaurus.config.js
+++ b/web/docusaurus.config.js
@@ -137,8 +137,8 @@ const config = {
       }
     ],
     './plugins/mui',
-    './plugins/analyze',
-  ],
+    process.env.ENABLE_BUNDLE_ANALYZE === 'true' ? './plugins/analyze' : undefined,
+  ].filter(Boolean),
 
   presets: [
     [

